### ISSUE TITLE
How to get a reference to layers loaded from a web scene?

### ISSUE THREAD

**Issue by mvrtich** [Esri/react-sceneview#38]
Hi,
building on simple example as this:
```
import React from 'react';
import { render } from 'react-dom';
import { SceneView, Scene } from 'react-sceneview';

render(
  <SceneView id="sceneview">
    <Scene portalItem={{ id: '34859cee6739438d93262a5aa91bf834' }} />
  </SceneView>,
  document.getElementById('root'),
);
```

how to get a reference to a webscene loaded as a portalItem, which I need to read all the layers coming with a webscene itself to inicialize my own components (eg. filtering or rendering settings).

Thank you,
Matej

**Comment by lisastaehli** [comment]
Hi Matej,

Thanks for your request. We will look into adding an `onLoad` property to the webscene so you can access the layers from that callback.

For now you would have two options to work around this:
1. Load the layers from the webscene separately.
2. Use the `window.sceneViews.sceneview.view.map.layers.items.getItemAt(index)` hack to access the layers individually (replace `index` with your own values)

Hope that helps,

Lisa

**Comment by mvdbster** [comment]
Hi @mvrtich. I've just published version `0.1.41`, which adds `onLoad` callbacks to both the `SceneView` and `Scene` component, returning a reference to the Esri JS API sceneview and webscene respectively. In other words, you can do:

```
<SceneView id="sceneview">
    <Scene
      portalItem={{ id: '34859cee6739438d93262a5aa91bf834' }}
      onLoad={webscene => handleWebsceneLoad(webscene)}
    />
  </SceneView>
```
